shard.yml

```diff
@@ -1,29 +1,29 @@
 name: lucky_flow
-version: 0.9.1
+version: 0.9.2
 
 authors:
   - Paul Smith <paulcsmith0218@gmail.com>
 
-crystal: ">= 1.4.0"
+crystal: "~> 1.4"
 
 license: MIT
 
 dependencies:
   selenium:
     github: crystal-loot/selenium.cr
-    version: ~> 0.12
+    version: ">= 0.12.1, < 0.13"
   webdrivers:
     github: crystal-loot/webdrivers.cr
-    version: ~> 0.4
+    version: ">= 0.4.3, < 0.5"
```

Comment:
Same here, just want to verify

Reply:
Same here. It needs to be at least 0.4.3 to work. 0.4.2 doesn't